JQuery.js

```diff
@@ -0,0 +1,21 @@
+
+var newThing = $("<li></li>");
+
+$('#name-form').on('submit', function(e) {                      
+  e.preventDefault();
+  var $name = $('#name-form input').val();
+  console.log($name);
+
+  $('#name-form input').val(''); 
+  $('ul').append('<li>' + "<button>"  + "X"+ "</button>"+ $name + '</li>');
```

Comment:
Since you've gone to the trouble of declaring ```var newThing = $('<li></li>');```, we can append that to the ```ul``` first, and then change the inner HTML of that list item.